src/NovakSolutions/Infusionsoft/Service/Traits/ListTrait.php

```diff
@@ -42,7 +42,7 @@ public static function find(array $criteria = [], $limit = null, $offset = null,
 
         foreach($criteria as $criteriaFieldName => $criterion){
             if(!in_array($criteriaFieldName, static::$findByFields)){
-                throw new FindException("Invalid field name: " . $criteriaFieldName . ' in service ' . (new \ReflectionClass(self))->getShortName());
+                throw new FindException("Invalid field name: " . $criteriaFieldName . ' in service ' . (new \ReflectionClass(self::class))->getShortName());
```

Comment:
This may need to be static::class instead of self::class.  Other wise "Late Static Binding" may or may not return the correct class if someone extends the class (or trait?).  But, I don't know for sure.  Just an FYI